src/UniversalTelegramBot.cpp

```diff
@@ -287,35 +328,36 @@ bool UniversalTelegramBot::getMe() {
 * (Argument to pass: the last+1 message to read)             *
 * Returns the number of new messages           *
 ***************************************************************/
-int UniversalTelegramBot::getUpdates(long offset)  {
-
-  if (_debug) Serial.println("GET Update Messages");
+int UniversalTelegramBot::getUpdates(long offset)  
+{
+  DEBUG_BOT("[BOT]GET Update Messages");
 
-  String command = "bot"+_token+"/getUpdates?offset="+String(offset)+"&limit="+String(HANDLE_MESSAGES);
+  String command = "bot"+_token+"/getUpdates?offset="+String(offset)+"&limit="+String(HANDLE_MESSAGES)+"&allowed_updates=[""message""]";
```

Comment:
Note here that I am limiting the update types to messages. This is because originally the way this library is implemented, it expects to receive messages. The telegramMessage data-structure reflects this.